src/test/TestUtils/TestProjectFixture.cs

```diff
@@ -297,6 +298,7 @@ private void ValidateRequiredDirectories(RepoDirectoriesProvider repoDirectories
             }
 
             storeArgs.Add($"/p:MNAVersion={_repoDirectoriesProvider.MicrosoftNETCoreAppVersion}");
+            storeArgs.Add($"/p:NETCoreAppFramework={_framework}");
```

Comment:
I think you end up passing this in twice, here and at line 280.

Reply:
I believe I still need to set the property so that it is correctly set in the msbuild property, but perhaps not I didn't verify that directly I just ensured we passed it everywhere we pass MNAVersion.